### ISSUE TITLE
There are cases of incorrect comment-out output in tsx

### ISSUE THREAD

**Issue by y-temp4** [kawamataryo/suppress-ts-errors#21]
## Error details

At [suppressTsErrors.test.ts](https://github.com/kawamataryo/suppress-ts-errors/blob/main/src/lib/__tests__/suppressTsErrors.test.ts), let us consider the following test case.

```tsx
function tsxFunc(num: number) {
  return (
    <div id={1}>
      <div>{num.map(n => n)}</div>
      <div>{num.map(n => n)}</div>
    </div>
  )
}
```

In this case, the following `expectedText` is expected, but unfortunately, this code does not work correctly as tsx.

```tsx
function tsxFunc(num: number) {
  return (
    {/*
     // @ts-expect-error TS7026 */}
    <div id={1}>
      // @ts-expect-error TS7026
      <div>{num.map(n => n)}</div>
      {/*
       // @ts-expect-error TS7026 */}
      <div>{num.map(n => n)}</div>
    {/*
     // @ts-expect-error TS7026 */}
    </div>
  )
}
```

I think the correct output should be as follows.

```tsx
function tsxFunc(num: number) {
  return (
    // @ts-expect-error TS7026
    <div id={1}>
      {/*
       // @ts-expect-error TS7026 */}
      <div>{num.map((n) => n)}</div>
      {/*
       // @ts-expect-error TS7026 */}
      <div>{num.map((n) => n)}</div>
    </div>
  );
}
```

**Comment by kawamataryo** [comment]
Thank you for the issue.

There seems to be an error in the logic for adding comments to tsx.
I will correct it.

**Comment by kawamataryo** [comment]
@y-temp4 

Sorry I'm late 🙏
I fixed it in #28.

If you have any more problems, please Issue.
